core/trino-spi/pom.xml

```diff
@@ -509,6 +509,16 @@
                                     <code>java.method.visibilityIncreased</code>
                                     <new>method int io.trino.spi.block.ByteArrayBlock::getRawValuesOffset()</new>
                                 </item>
+                                <item>
```

Comment:
Booooo! But .. oh well.. I guess we have to

Reply:
Yeah.